kustomize/network/calico-etcd/patch_image_name.yaml

```diff
@@ -8,5 +8,5 @@ spec:
     spec:
       containers:
       # Change the value of image field below to your controller image URL
-      - image: magicsong/ks-network:90fa3885
+      - image: magicsong/ks-network:f22e8ea9
```

Comment:
push image to kubespheredev/ks-network, I've already created that repo

Reply:
done. but editing here is useless, refer to <https://github.com/kubesphere/kubesphere/pull/692/commits/d3e694c6f088331ae876ffa40b3a0cecdfca8681>